crates/theme/src/schema.rs

```diff
@@ -47,6 +60,9 @@ pub struct ThemeContent {
     pub name: String,
     pub appearance: AppearanceContent,
     pub style: ThemeStyleContent,
+    /// Background style ("opaque", "transparent" or "blurred").
+    #[serde(default, rename = "window.background")]
+    pub window_background: Option<String>,
```

Comment:
We should have a separate enum for the `WindowBackgroundContent`, similar to what we have for `AppearanceContent`.

This will provide better autocomplete with the JSON schema.